modules/ROOT/pages/_partials/push/handling-push-notifications.adoc

```diff
@@ -52,26 +52,22 @@ Cordova::
 --
 // end::excludeDownstream[]
 
-. For plain Cordova, install the push plugin:
+. Install the push plugin:
```

Comment:
just noticed that we've already asked users to do this earlier at https://deploy-preview-719--aerogear-docs-preview.netlify.com/aerogear/latest/push-notifications/#setting-up-sdk-push

Reply:
If the question is if we can remove it, I'd say yes. I just leave it there because it was already there. Do you wanna I remove it?

Reply:
Also think this can be removed.

Reply:
Removed!